docs/source/models/causes/neonatal/index.rst

```diff
@@ -128,28 +137,36 @@ unit time.
     digraph NN_decisions {
         rankdir = LR;
         lb [label="live birth", style=dashed]
-        nn_alive [label="neonate survived"]
-        nn_dead [label="neonate died"]
-
-        lb -> nn_alive  [label = "1 - mr"]
-        lb -> nn_dead [label = "mr"]
+        enn_alive [label="neonate survived\nfirst 7 days"]
+        enn_dead [label="neonate died\nduring first 7 days"]
+        lnn_alive [label="neonate survived\nfirst 28 days"]
+        lnn_dead [label="neonate died\nduring first 28 days"]
+
+        lb -> enn_alive  [label = "1 - enn_mr"]
+        lb -> enn_dead [label = "enn_mr"]
+        enn_alive -> lnn_alive [label = "1 - lnn_mr"]
+        enn_alive -> lnn_dead [label = "lnn_mr"]
```

Comment:
This graph seems excessively wide -- not sure if there's a good way to fix it

Reply:
I found some hack but there must be a better way
```
    digraph NN_decisions {
        graph [size="7,5"]
        rankdir = LR;
        ...
```